episodes/31-multi-files.md

```diff
@@ -26,6 +26,99 @@ bash script that calls our already-existing Python script in a for-loop, or
 (2) modifying our Python script to read multiple files in a for-loop.
 We will try both methods and compare the two.
 
+## Saving plot before branching
+
+Our code currently pops up the figure instead of saving it.
+Most of the time we will want to save the figure in any case but it will also help with the flow or our program with loops. With the current configuration, when we write the for loops it will pause each time it pops up a figure and wait for us to close the viewer.
```

Comment:
```suggestion
Most of the time we will want to save the figure as a separate file instead of only viewing it but saving instead of viewing will also help with the flow or our program's for loops. With the current configuration, when we run the for loop it will pause each time it pops up a figure and wait for us to close the viewer.
```